pep-0544.txt

```diff
@@ -1377,7 +1377,7 @@ Support adapters and adaptation
 -------------------------------
 
 Adaptation was proposed by :pep:`246` (rejected) and is supported by
-``zope.interface``, see https://docs.zope.org/zope.interface/adapter.html.
+``zope.interface``, see `https://docs.zope.org/zope.interface/adapter.html <https://web.archive.org/web/20160802080957/https://docs.zope.org/zope.interface/adapter.html>`_.
```

Comment:
This change is good. The original link is a 404.

Reply:
Just to verify, should I re-wrap this text manually, or is it good as-is?

Reply:
If CI is happy with it, I am happy.

Reply:
```suggestion
``zope.interface``, see `the Zope docs
<https://web.archive.org/web/20160802080957/https://docs.zope.org/zope.interface/adapter.html>`_.
```

This breaks more cleanly, and takes the opportunity to use proper descriptive inline link text per PEP 12, if we're going to change this to begin with.

Reply:
That sounds like a good idea. I have pushed a commit with improved link text.